themes/ddbasic/sass/module/_ding_nodelist.scss

```diff
@@ -0,0 +1,1101 @@
+      line-height: 32px;
+      color: $white;
+      content: '\f054';
+      text-align: center;
+      font-family: 'FontAwesome';
+    }
+
+    a {
+      display: none;
+    }
+  }
+
+  .slide-open {
+
+    .expand {
+      right: 10px;
+      left: auto;
+
+      &:after {
+        content: '\f067';
+      }
+    }
+
+    .image img {
+      opacity: 1;
+    }
+  }
+
+  // Image
+
+  .image {
+
+    a {
+      position: absolute;
+      top: 0;
+      left: 0;
+      z-index: 0;
+      display: block;
+      background-color: $black;
+
+      // Media small
+
+      @include small() {
+        width: 310px;
+        height: 260px;
+      }
+
+      // Media medium
+
+      @include medium() {
+        width: 400px;
+        height: 300px;
+      }
+
+      // Media large-x-large
+
+      @include large-x-large() {
+        width: 450px;
+        height: 340px;
+      }
+    }
+
+    img {
+      opacity: .25;
+      min-width: 100%;
+      width: auto;
+      height: 100%;
+    }
+  }
+
+  // Data block
+
+  .data {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    padding: 10px;
+    color: $white;
+
+    span,
+    .event-fee {
+      margin-right: 5px;
+    }
+
+    a {
+      color: $white;
+    }
+  }
+
+  // Caption
+
+  .caption {
+    overflow: hidden;
+
+    h3 {
+      max-height: 30px;
+      margin: 5px 0;
+
+      // Media small-medium
+
+      @include small-medium() {
+      }
+    }
+  }
+
+  // Library info
+
+  .library {
+    overflow: hidden;
+    max-height: 24px;
+
+    > div {
+      float: left;
+    }
+  }
+}
+
+// Ding nodelist single item
+//==========================================================================
+
+.ding_nodelist-single {
+  overflow: hidden;
+  background-color: $white;
+
+  // Image
+
+  .article_image,
+  .event-image {
+
+    a {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background-repeat: no-repeat;
+      background-size: cover;
+      width: 100%;
+      height: 100%;
+    }
+
+    img {
+      display: block;
+      width: 100%;
+    }
+  }
+
+  // Item
+
+  .item {
+    position: relative;
+    overflow: hidden;
+
+    // Media from-small
+
+    @include from-small() {
+      height: 381px;
+    }
+
+    // Date
+
+    .event-label,
+    .event-time {
+
+      // Media small
+
+      @include small() {
+        visibility: hidden;
+      }
+    }
+
+
+  }
+
+  // Title
+
+  h3 {
+    margin: 0 0 10px 0;
+
+    a {
+      display: block;
+      color: $white;
+
+      // Media large-x-large
+
+      @include large-x-large() {
+      }
+    }
+  }
+}
+
+
+// Nodelist slider
+//==========================================================================
+
+.ding_nodelist-slider {
+  position: relative;
+  background-color: $white;
+  padding-top: 10px;
+
+  .item {
+    min-height: 122px;
+  }
+
+  // Title
+
+  .node {
+    background: none;
+    color: $black;
+    padding: 0 !important;
```

Comment:
if `!important` is necessary here please add a comment expalining why. We generally try to avoid that.